workflows/dgd_fusion_annotator_wf.cwl

```diff
@@ -0,0 +1,70 @@
+cwlVersion: v1.2
+class: Workflow
+id: dgd-fusion-annotator-wf
+label: DGD Fusion Annotator Workflow
+doc: "Formats and annotates DGD fusion files"
+
+
+inputs:
+  # multi
+  output_basename: { type: string, doc: "Basename for the output TSV file" }
+  # update gene symbols
+  hgnc_tsv: { type: 'File?', doc: "Gene name database TSV file from HGNC. i.e. hgnc_complete_set.txt. If not provided, updated will be skipped" }
+  fusions_tsv: { type: File, doc: "Custom Fusions TSV file, i.e. fusion-dgd.tsv.gz" }
+  old_symbol: { type: 'string?', doc: "Column name for the old gene symbol(s) in the HGNC TSV. Set to override script defaults" }
+  new_symbol: { type: 'string?', doc: "Column name for the new gene symbol(s) in the HGNC TSV. Set to override script defaults" }
+  update_columns: {type: 'string?', doc: "Space-separated column names from the Fusions TSV where to update gene names (e.g. -u foo bar blah). Set to override script defaults" }
+  # fusion standardization
+  caller: { type: [{type: enum, name: caller, symbols: ["STARFUSION", "ARRIBA", "DGD", "CUSTOM"]}], doc: "Caller used to produce input",
+    default: "DGD" }
+  # annotate fusion
+  genome_tar: {type: 'File', doc: "STAR-Fusion CTAT Genome lib", "sbg:suggestedValue": {
+      class: File, path: 62853e7ad63f7c6d8d7ae5a8, name: GRCh38_v39_CTAT_lib_Mar242022.CUSTOM.tar.gz}}
+  genome_untar_path: {type: ['null', string], doc: "This is what the path will be when genome_tar is unpackaged", default: "GRCh38_v39_CTAT_lib_Mar242022.CUSTOM"}
+  col_num: {type: ['null', int], doc: "column number in file of fusion name, use 24 for arriba v1.1, 30 for v2, 1 for DGD", default: 1}
```

Comment:
lets just go with `int?` notation.

Reply:
Ah, yeah, that was a bit of copy/paste since that tool is from the RNAseq wf...I was tempted to make the wf a sub module, but seems heavy-handed for just one tool. @sickler-alex also faced that issue...